src/Node/AbstractNode.php

```diff
@@ -186,11 +185,11 @@ public function cyclomaticComplexity(): int
         $ccn = 0;
 
         foreach ($this->classesAndTraits() as $classLike) {
-            $ccn += $classLike['ccn'];
+            $ccn += $classLike->ccn;
```

Comment:
Is there really a measurable performance benefit in using a public property over a method call to a public method that returns the value of a private property?

Reply:
Yes, doing so in https://github.com/sebastianbergmann/php-code-coverage/pull/1105